migrations/helpers/OutputLogTemplate.js

```diff
@@ -0,0 +1,64 @@
+const logger = require('../../src/logger/');
+
+
+class OutputLogTempalte {
+	constructor({
+		total,
```

Comment:
why only use one object param instead of single params here?

Reply:
You can read the parameter as options. Here you can set exactly the options that are relevant for the use case.